src/components/common.js

```diff
@@ -62,8 +62,15 @@ export const Clickable = ({ href, as = (!!href ? 'a' : 'div'), disabled, tooltip
     ...props
   }, [children])
 
+  // If there's only one child which is an icon, and no aria-label, and the user hasn't explicitly set useTooltipAsLabel to false,
+  // then use the tooltip as the label rather than the description.
```

Comment:
Technically, it's using the tooltip as either a label or a description, but not both. But it's not taking an existing description and turning it into a label...

You're absolutely right that this is confusing language! I'll clean it up.

Reply:
The wording has been updated! Now includes a better description of why this check is even necessary, and frames the logic in a positive way, rather than all the negation I had before